lib/led-matrix/include/LEDMatrix.h

```diff
@@ -0,0 +1,37 @@
+/* LED Matrix Header */
+#pragma once
+#include <mbed.h>
+#include <CANMsg.h>
+class LEDMatrix
+{
+  public:
+	// Constructor: An LED Matrix is defined by the pins it is connected to
```

Comment:
something you might want to look into are Doxygen comments. They are a standard way of commenting your functions. However the comments you have here are good too https://www.doxygen.nl/manual/docblocks.html